.github/allowed-labels.yml

```diff
@@ -115,72 +126,30 @@
   color: "00FFFF"
   description: "This work needs input from the reporter"
   from_name: "needs input"
-- name: status/needs-research
+- name: research-spike
   color: "00FFFF"
   description: "This work needs research by the resolver"
-  from_name: "needs research"
+  from_name: "status/needs-research"
```

Comment:
I don't  think we need a `status/needs-research` for a PR. If a PR needs research, then it should be closed until it can be reopened with confidence.

Reply:
@ford-at-aws What if it's an external PR? I could see a case where we want to test out a  large change set before merging. That said, we don't need a label for that, we simply withhold the approval.

Reply:
Let's merge for now and have a conversation about what we want from our research indicating labels.